### PR TITLE
fix(Example): use pseudo class for active link

### PR DESCRIPTION
**Pull request by kevmoo**
Also remove a superfluous comment

### CODE REVIEW

File: examples/hacker_news_pwa/web/index.html [hunk 1]
```diff
@@ -59,7 +59,7 @@
     nav a:hover {
       color: #fff;
     }
-    nav a.active {
+    nav a:active {
```
Comment (leonsenft) [inline examples/hacker_news_pwa/web/index.html]: Did you test this? This wasn't a typo. This is a specific class added by the routerLink directive, that the router adds/removes when the link is active.

https://github.com/dart-lang/angular/blob/843cb0fb5972b6a335a07ca65fd54273fe08a328/examples/hacker_news_pwa/lib/app_component.html#L6

File: examples/hacker_news_pwa/web/index.html [hunk 2]
```diff
@@ -112,7 +107,7 @@
         <a href="https://webdev.dartlang.org/angular">
           <img src="/images/logo.svg" alt="AngularDart logo" />
         </a>
-        <a href="">top</a>
+        <a href="" class="active">top</a>
```
Comment (leonsenft) [inline examples/hacker_news_pwa/web/index.html]: This shouldn't be here.

1. You moved the style from this file, to app_component.css. By the time that file is loaded this static HTML will have been replaced anyways.
2. Even if you moved the style back, we don't want to assume users loaded the app at /. They might have loaded /ask, so highlighting this link preemptively would be incorrect.

File: examples/hacker_news_pwa/lib/app_component.css [hunk 3]
```diff
@@ -6,3 +6,8 @@
   width: 100%;
   max-width: 800px;
 }
+nav a.active {
```
Comment (leonsenft) [inline examples/hacker_news_pwa/lib/app_component.css]: Good idea moving this here!

**Comment by leonsenft** [comment]
LGTM. Seeing as I can't squash before rebasing, would you mind doing so to keep our change log clean since half of these commits undo a prior one?

**Comment by kevmoo** [comment]
Sure!

**Comment by kevmoo** [comment]
Uploaded fewer commits – but nothing to rebase

**Comment by matanlurey** [comment]
@kevmoo Do you mind syncing your own PRs?

**Comment by kevmoo** [comment]
> Do you mind syncing your own PRs?

Done

**Pull request closed without merging**